waterlog/src/Components/MonthlyCostsReports.js

```diff
@@ -2,57 +2,67 @@ import React from 'react'
 import { Bar, defaults } from 'react-chartjs-2';
 
 const MonthlyCostsReports = (props) => {
-    if (props.props.dataPoints) {
-        var dataY = props.props.dataPoints.map(a => Math.round(a.y));
-        var data = {
-            labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
-            datasets: [
-                {
-                    label: 'rands',
-                    data: dataY,
-                    fill: true,
-                    borderColor: 'rgba(255,0,0,0)',
-                    backgroundColor: 'rgba(255,0,0,1)'
-                }
-            ]
+  if (props.props.monthlyCost.dataPoints) {
+    var dataY = props.props.monthlyCost.dataPoints.map(a => Math.round(a.y));
+    const forecast = props.props.forecastMonthly;
+    var data = {
+      labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+      datasets: [
+        {
+          label: 'rands',
+          data: dataY,
+          fill: true,
+          borderColor: 'rgba(255,0,0,0)',
+          backgroundColor: 'rgba(255,0,0,1)'
+        },
+        {
+          label: 'forecast',
+          data: forecast,
+          fill: true,
+          borderColor: 'rgb(0,191,255)',
+          backgroundColor: 'rgb(0,191,255)'
```

Comment:
line 15-16 is rgba
stay with it. 
change 23-23 to it

Reply:
changed to rgba.